src/current/_includes/v24.1/cdc/disable-replication-ttl.md

```diff
@@ -0,0 +1,26 @@
+{% include_cached new-in.html version="v24.1" %} Use the `ttl_disable_changefeed_replication` table storage parameter to prevent changefeeds from sending `DELETE` messages issued by row-level TTL jobs for a table. Include the storage parameter when you create or alter the table. For example:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+CREATE TABLE tbl (
+  id UUID PRIMARY KEY default gen_random_uuid(),
+  value TEXT
+) WITH (ttl_expire_after = '3 weeks', ttl_job_cron = '@daily', ttl_disable_changefeed_replication = 'true');
+~~~
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+ALTER TABLE events SET (ttl_expire_after = '1 year', ttl_disable_changefeed_replication = 'true');
+~~~
+
+You can also widen the scope to the cluster by setting the `sql.ttl.changefeed_replication.disabled` [cluster setting]({% link {{ page.version.version }}/cluster-settings.md %}) to `true`. This will prevent changefeeds from emitting deletes issued by TTL jobs on a cluster.
```

Comment:
nit: maybe say "all TTL jobs"?

Reply:
Done! And yes, I did still need to remove the session var from the v23.2 include file. Sorry about that, should be fixed now!